theano/tensor/nnet/opt.py

```diff
@@ -42,6 +42,8 @@ def local_inplace_sparse_block_gemv(node):
         copy_stack_trace(node.outputs[0], new_node)
         return [new_node]
     return False
+
```

Comment:
I've added these lines, because of flake8 warnings (might be some misconfiguration on my side), I thought the spaces were removed by doing the rebase.